apps/api-documenter/src/cli/MarkdownAction.ts

```diff
@@ -20,23 +18,15 @@ export class MarkdownAction extends BaseAction {
   }
 
   protected async onExecute(): Promise<void> {
-    let configFilePath: string | undefined = path.join(process.cwd(), DocumenterConfig.FILENAME);
-
-    // First try the current folder
-    if (!FileSystem.exists(configFilePath)) {
-      // Otherwise try the standard "config" subfolder
-      configFilePath = path.join(process.cwd(), 'config', DocumenterConfig.FILENAME);
-      if (!FileSystem.exists(configFilePath)) {
-        console.warn(
-          `Unable to find ${DocumenterConfig.FILENAME} in the current folder or in a "config" subfolder`
-        );
-        configFilePath = undefined;
-      }
+    const documenterConfig: DocumenterConfig | undefined =
+      await DocumenterConfig.tryLoadFileFromDefaultLocationsAsync(process.cwd());
+    if (!documenterConfig) {
+      console.log(
+        `Did not find a ${DocumenterConfig.FILENAME} in the current folder or in a "config" subfolder`
```

Comment:
The previous wording seemed better:
```
`Unable to find ${DocumenterConfig.FILENAME} in the current folder or in a "config" subfolder`
```

Reply:
Originally this command didn't load the config file at all, so I'm changing the wording to not make it sound like not loading that file is an issue.